### PR TITLE
Style markers and intervals

### PR DESCRIPTION
**Pull request by 0x213F**

### CODE REVIEW

File: src/components/PlaybackApp/ChildProgressBar/ChildProgressBar.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ function ChildProgressBar(props) {
         editable = props.editable;
 
   const intervalWidthDuration = interval.endPosition - interval.startPosition,
-        intervalWidth = intervalWidthDuration / duration * 400;
+        intervalWidth = intervalWidthDuration / duration * 100;
```
Comment (op1346) [inline src/components/PlaybackApp/ChildProgressBar/ChildProgressBar.js:19]: Just wondering why it was 400 before?

File: src/components/PlaybackApp/ChildProgressBar/ChildProgressBar.js [hunk 2]
```diff
@@ -52,25 +52,26 @@ function ChildProgressBar(props) {
    */
   return (
     <div className={styles[progressClass]}
-         style={{width: `${intervalWidth}px`}}
+         style={{width: `${intervalWidth}%`}}
```
Comment (op1346) [inline src/components/PlaybackApp/ChildProgressBar/ChildProgressBar.js:55]: yasss I've heard that using percentages is so much better!

**Review by op1346 — APPROVED** [review]
I love the clean-up on this one!

File: src/components/PlaybackApp/ChildProgressBar/ChildProgressBar.js (hunk 1 above)
Comment (0x213F) [reply src/components/PlaybackApp/ChildProgressBar/ChildProgressBar.js:19]: The bar width was `400px` total. So I kept the same math but instead made it out of `100` and changed the postfix from `px` to `%` 😄

**Pull request merged**